Sources/TuistKit/Services/TestService.swift

```diff
@@ -486,7 +490,11 @@ final class TestService { // swiftlint:disable:this type_body_length
 
         let testSchemes = schemes
             .filter {
-                $0.testAction.map { !$0.targets.isEmpty } ?? false
+                $0.testAction.map { !$0.targets.isEmpty } ?? false || !self.testActionTargets(
```

Comment:
Maybe it'd be better to remove the check: 
```
$0.testAction.map { !$0.targets.isEmpty }
```

Since the call to `testActionTargets` will also check this

Thoughts?

Reply:
yes, let's do it.